dashboard/app/models/user.rb

```diff
@@ -720,7 +720,8 @@ def all_advertised_scripts_completed?
 
   def completed?(script)
     user_script = user_scripts.where(script_id: script.id).first
-    user_script.try(:completed_at) || (user_script && next_unpassed_progression_level(script).nil?)
+    return false unless user_script
+    user_script.completed_at || next_unpassed_progression_level(script).nil?
```

Comment:
are we confident there is no context in which we would still want to use `.try`?